### PR TITLE
Refactor/themed logo

### PR DESCRIPTION
**Pull request by petrKavulok**
Preparation work for dark themed logo. Renaming logo files.

TODO:
- [x]  asab-webui update when https://github.com/TeskaLabs/asab-webui/pull/375 is finished
- [x] Changelog updates

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,7 +29,15 @@ const ConfigDefaults = {
 		fallbackLng: 'en',
 		supportedLngs: ['en', 'cs'],
 		debug: false,
-		nsSeparator: false
+		nsSeparator: false,
+	},
+	brandImage: {
```
Comment (Pe5h4) [inline src/index.js:34]: @petrKavulok Would it work also even without defining these defaults as it was before?

Comment (Pe5h4) [reply src/index.js:34]: @petrKavulok Btw. I would assume that SeaCat Auth UI will use also small logo maybe in the future

**Review by Pe5h4 — APPROVED** [review]
@petrKavulok Dont forget for submodule update

File: src/index.js (hunk 1 above)
Comment (petrKavulok) [reply src/index.js:34]: @Pe5h4 it does work even when nothing is defined. it returns the same default for both dark & light versions.

Screenshot 2023-05-11 at 9 28 13
Screenshot 2023-05-11 at 9 28 24

**Pull request merged**